cachi2/core/package_managers/yarn/resolver.py

```diff
@@ -275,6 +277,59 @@ def log_for_locator(msg: str, *args: Any, level: int = logging.DEBUG) -> None:
 
         return _ResolvedPackage(locator, name, version, checksum)
 
+    def _generate_purl_for_package(self, package: _ResolvedPackage, project: Project) -> str:
```

Comment:
shall we make this method a `staticmethod` since it doesn't interact with `self` in any way?